db/migrate/20230816032328_create_comments.rb

```diff
@@ -0,0 +1,13 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :user_id
+      t.integer :post_id
+      t.string :text
+
+      t.timestamps
+    end
```

Comment:
- [ ] As required for the project to correctly match the the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) give to you in the project requirements, kindly ensure that the foreign key to the `users` table is named `author_id`. I suggest you use the same approach, I mentioned in the previous comment even if it is added in a different file i.e. ``t.references :author, foreign_key: { to_table: 'users' }``